_posts/2017-09-25-bagwell-award-2017.md

```diff
@@ -0,0 +1,39 @@
+---
+category: news
+title: "Josh Suereth receives community award"
+---
+
+We are pleased to announce that the Phil Bagwell Memorial Scala Community Award for 2017 has been awarded to [Josh Suereth](https://twitter.com/jsuereth).  The award was presented in England's Lake District at [Scala World 2017](https://scala.world).
+
+Josh has been a prominent member of the Scala community ever since the early
+days of the language's commercial adoption.  He is known to Scala users as:
+
+* author of [_Scala in Depth_](https://www.manning.com/books/scala-in-depth)
+  and co-author of [_sbt in Action_](https://www.manning.com/books/sbt-in-action)
+* maintainer of the [sbt](http://www.scala-sbt.org) build tool
+  for several years at Typesafe (the company now known as Lightbend)
+* leader or contributor to numerous other open source Scala projects
+  including scala-arm, pickling, and assorted sbt plugins
+* cohost of the [Scalawags podcast](http://scalawags.tv)
+  (and its predecessor, The Scala Types)
+* a friendly and tireless conference-goer and presenter (and
+  organizer of the Scala Lift Off unconference in Virginia in 2009)
+* a positive force on nearly every online medium devoted to
+  Scala -- mailing lists, chat rooms, Twitter, you name it --
+  answering any and all questions with tact and humor
+
+Here's a
+[three minute video](https://www.youtube.com/watch?time_continue=2&v=0sLixaObVwA)
+of friends and colleagues paying tribute to Josh.  The video was shown
+when the award was presented.
+
+Josh's surname is pronounced "Suereth".
+
+The Bagwell Award is named in honor of Phil Bagwell, who passed away in 2012.  In a [2012 blog post](https://www.typesafe.com/blog/rip-phil-bagwell), Martin Odersky remembers Phil and his special place in the Scala community.
+
+Past winners of the award:
+
+* 2016: [Erik Osheim](https://www.scala-lang.org/news/2016/10/26/bagwell-award-2016.html)
+* 2015: [Bill Venners](http://scala-lang.org/news/2015/06/25/bagwell-award-2015.html)
+* 2014: [Lalit Pant](https://kojoenv.wordpress.com/2014/09/27/phil-bagwell-award/)
+* 2013: [Dick Wall](https://twitter.com/dickwall)
```

Comment:
Someone told me that Debasish won the award one year. Did that really happen? Was it the first year?

Reply:
the award didn't formally exist before 2013, but at Scala Days 2012 in London, I remember @odersky getting up and recognizing Nathan Hamblen for his community work. I think that was essentially a pre-Bagwell Award Bagwell Award, and I've wondered before if it should be listed here.

perhaps Debasish got some similar recognition and I'm just not aware.